pkgs/tools/nix/nix-melt/default.nix

```diff
@@ -1,5 +1,6 @@
 { lib
 , rustPlatform
+, fetchpatch2
```

Comment:
This line can be dropped.

Reply:
Oh indeed! Sorry to have missed that haha, I was trying to fetch an upstream patch...